components/SoundPlayer.tsx

```diff
@@ -15,7 +15,7 @@ import { getInstrumentIdByValue, instruments } from "midi-instruments";
 import { VISUALIZER_MODE } from "@enums/visualizerMessages";
 import webMidi from "webmidi";
 import Tone from "tone";
-import { DEFAULT_FIRST_KEY, DEFAULT_LAST_KEY } from "@config/piano";
+import { getDefaulRange, setDefaultKeyRange } from "@config/piano";
```

Comment:
I think this is a typo: 

```suggestion
import { getDefaultRange, setDefaulttKeyRange } from "@config/piano";
```

Reply:
oops! good catch. fixed.